test_openai.py

```
@@ -0,0 +1,3 @@
+import openai
```

Ruff (0.12.2)

`openai` imported but unused

Remove unused import: `openai`

(F401)